lib/cc/engine/analyzers/violation.rb

```diff
@@ -45,7 +64,9 @@ def sorted_hashes
         end
 
         def other_sexps
-          @other_locations ||= sorted_hashes.drop(1)
+          sorted = sorted_hashes
```

Comment:
@wfleming Actually, that implementation doesn't work. The `sexp` doesn't have a location attached to it, so multiple sexps could be equal.